var/spack/repos/builtin/packages/metall/package.py

```diff
@@ -0,0 +1,23 @@
+# Copyright 2013-2020 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+
+class Metall(CMakePackage):
+    """An allocator for persistent memory"""
+
+    homepage = "https://github.com/LLNL/metall"
+    git      = "git@github.com:LLNL/metall.git"
```

Comment:
```suggestion
    git      = "https://github.com/LLNL/metall.git"
```
I've seen problems with the SSH-style download links in the past.

Reply:
Thank you for the suggestion!